### ISSUE TITLE
Comment style: ## vs. #

### ISSUE THREAD

**Issue by ypid** [debops/debops#1620]
I used `##` for a long time so you find this in DebOps. My idea of the two comment signs was to differentiate between commented out code (`#`) and comments in natural language. I recently came to the conclusion that this is not good style because there should not be commented out code (we have git for that) and because editor support for this is bad. Also, Python linters force one `#` and a space.

Consider:

```bash
## Human comment.
# my_commented_out_var=23
actual_code()
```
vs.

```bash
# Human comment.
actual_code()
```

Which do you favor?

**Comment by drybjed** [comment]
I think that we should stick to the style used by a given project or configuration file or language we are integrating within DebOps. When I create new template for a particular configuration file, I try to recreate it as closely as possible so that people who know that given application don't have to learn some new style used in DebOps. That also means that linters for a given language work as expected. So, if `pycodestyle` wants one `#` and a space, we should go with that.

**Comment by ypid** [comment]
I agree. I was more thinking about the parts that we have more freedom over, like scripts we maintain and so on.
